### PR TITLE
Added metrics docs

### PR DESCRIPTION
**Pull request by jjcollinge**
# Description

Added docs for the metrics endpoint that was in [this PR](https://github.com/dapr/dapr/pull/962).

## Issue reference

Please reference the issue this PR will close: [#959](https://github.com/dapr/dapr/issues/959)

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [x] Extended the documentation

### CODE REVIEW

**Comment by youngbupark** [comment]
Thanks for all your work! :)

Can you please change the doc structure ? concepts/observability/metrics.md
Trace/logs/metrics will belong to `observability`

Also please add [how-to doc](https://github.com/dapr/docs/tree/master/howto) to install and use prometheus.

**Comment by jjcollinge** [comment]
Yh no problem. I won’t be able to get to this until next week though.

**Comment by jjcollinge** [comment]
We should wait on [1053](https://github.com/dapr/dapr/issues/1053) to be closed before merging this.

**Comment by youngbupark** [comment]
@jjcollinge Can you please double-check if there is any document referring to distributed-tracing/README.md. As I remember, there are some documents referring to trace README.md. 

You may need to correct those links or to revert trace README.md and change it later in separate PR.

I recommend the later one to keep it simple :)

Thanks for your contribution!

File: howto/observe-metrics-with-prometheus/README.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+```
+$ kubectl get svc
+
+NAME          TYPE       CLUSTER-IP    EXTERNAL-IP  PORT(S)
+nodeapp-dapr  ClusterIP  10.0.163.251  <none>       80/TCP,50001/TCP,9090/TCP
+...
+```
+
+You can then add the services DNS to your Prometheus configuration.
+```yaml
```
Comment (youngbupark) [inline howto/observe-metrics-with-prometheus/README.md]: isn't there a way to annotate pod to make prometheus operator recognize metric endpoint automatically?

File: howto/observe-metrics-with-prometheus/README.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
+# Observe Metrics with Prometheus
+Dapr exposes a Prometheus metrics endpoint you can use to collect time-series
+data relating to the execution of the Dapr runtime itself.
+
```
Comment (youngbupark) [inline howto/observe-metrics-with-prometheus/README.md]: I would add contents such as
* [Setup Prometheus Locally](#setup-prometheus-locally)
* [Setup Prometheus on Kubernetes](#setup-prometheus-on-kubernetes)

File: howto/observe-metrics-with-prometheus/README.md (hunk 1 above)
Comment (youngbupark) [reply howto/observe-metrics-with-prometheus/README.md]: nevermind:)

File: concepts/observability/metrics.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# Metrics
+Dapr exposes a [Prometheus](https://prometheus.io/) metrics endpoint that you can scrape to gain
```
Comment (msfussell) [inline concepts/observability/metrics.md]: Is scrape the right word here?

File: concepts/observability/metrics.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+a greater understanding of how Dapr is behaving and to setup alerts for specific conditions.
+
+## Configuration
+The metrics endpoint is enabled by default, you can disable it by passing the command line argument
```
Comment (msfussell) [inline concepts/observability/metrics.md]: There are no good docs on daprd today. I raised issue https://github.com/dapr/docs/issues/378

File: concepts/observability/metrics.md (hunk 3 above)
Comment (youngbupark) [reply concepts/observability/metrics.md]: Yes scrape is the right term.

**Comment by youngbupark** [comment]
@jjcollinge can you fix the conflict ?

**Comment by jjcollinge** [comment]
@youngbupark these'll need updating given the latest implementation right?

File: concepts/observability/metrics.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+`--enable-metrics false` to daprd.
```
Comment (youngbupark) [inline concepts/observability/metrics.md]: now we need to use `--enable-metrics=false`  flag with =

**Comment by youngbupark** [comment]
> @youngbupark these'll need updating given the latest implementation right?

there is only one change for flag. other than that, everything is same. 

For K8s cluster, I would like to use helm chart to install prometheus. We can update this part iteratively once you merge this pr.

**Pull request merged**